source/_integrations/iron_os.markdown

```diff
@@ -0,0 +1,35 @@
+---
+title: ironOS
+description: Instructions on how to integrate ironOS-based Pinecil V2 devices with Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Local Polling
+ha_release: 2024.8
+ha_config_flow: true
+ha_codeowners:
+  - '@tr4nt0r'
+ha_domain: iron_os
+ha_integration_type: integration
+ha_platforms:
+  - sensor
+---
+
+The **ironOS** {% term integration %} seamlessly connects Home Assistant with PINE64's Pinecil V2 soldering irons, allowing for remote monitoring and control. This integration provides real-time updates on temperature, power, and various other settings and diagnostic information.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
```

Comment:
I think for some of descriptions it could be helpful to have a unit. (for example: is it in degrees Celsius? the duration: is it in minutes?)

Reply:
But the unit and precision depends on what the user sets in Home Assistant? For temperature it could be either Kelvin, Fahrenheit or Celsius?

Reply:
oh, never mind then. I thought it was specific to this sensor.